dao-user-reference.adoc

```diff
@@ -0,0 +1,192 @@
++
+2. _After October 2017_
++
+In October 2017, the Phase Zero plan went into effect. Subsequent contributions to Bisq were https://github.com/bisq-network/compensation/issues[documented on GitHub^] and votes were cast https://docs.google.com/spreadsheets/d/1xlXDswj3251BPCOcII-UyWlX7o7jMkfYBE-IZ5te5Ck[on this spreadsheet^] by stakeholders.
+
+The genesis BSQ distribution was the sum of both segments of past contributions.
+
+Ultimately, the genesis transaction issued https://explorer.bisq.network/[6,157,480 BSQ to 202 contributors^].
+
+You can verify the calculation of the final issuance amount yourself. The portion allotted for contributors from before October 2017 was 2.5 million BSQ. The remainder was for contributions made afterward, which are all documented on GitHub and the spreadsheet (both linked above).
+
+== Bisq DAO
+
+=== Technical Overview
+
+There are 2 key points to note about how the Bisq DAO works.
+
+1. **Every DAO action is an on-chain Bitcoin transaction.** Every DAO action requires BSQ, and since BSQ is bitcoin, every DAO action is also recorded on the Bitcoin blockchain. As noted above, this does NOT mean you should try to form your own BSQ transactions. But it does mean that every action you take on the Bisq DAO is immortalized on the Bitcoin blockchain, publicly verifiable, and virtually impossible to reverse.
+
+2. **Bitcoin provides transaction integrity, and Bisq's own P2P network provides storage.** DAO actions such as making proposals and voting include extra data such as text and links that don't belong on the Bitcoin blockchain. This extra data is stored on Bisq's own P2P network (literally, Bisq users' hard drives). A hash of that extra data is included in the `OP_RETURN` output of the corresponding Bitcoin transaction. Then, the Bitcoin transaction ID is added to the extra data and broadcast to peers on the Bisq network.
+
+You can visualize it like this:
+
+.How Bisq stores data
+image::bisq-dao-data-model.png[How Bisq stores data]
+
+Storing a hash of the extra data on the Bitcoin blockchain ensures any attempts to manipulate it will fail, since the hash of the manipulated data will change and then not match the hash recorded in the corresponding Bitcoin transaction.
+
+NOTE: There's a bit of a chicken-and-egg problem here, since the Bitcoin transaction ID can only be known after the Bitcoin transaction is made, and the hash of a data object can only be made once the Bitcoin transaction ID is known. To get around this, the data object is hashed with a blank string in place of the Bitcoin transaction ID, and the Bitcoin transaction ID is added afterward. Later, when the integrity of the data object needs to be validated, the Bitcoin transaction ID is replaced with a blank string before the data object is hashed.
```

Comment:
"is hashed with a blank string" - sorry, that info from myself was wrong. it is set to null instead an empty string.